### PR TITLE
table theming update

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/pineapple.theme [hunk 1]
```diff
@@ -78,6 +78,17 @@ function pineapple_theme_suggestions_form_alter(array &$suggestions, array $vari
...
+ * Implements hook_theme_suggestions_table_alter().
+ */
+function pineapple_theme_suggestions_table_alter(array &$suggestions, array $variables) {
+  if (isset($variables['attributes']['id'])) {
+    $table_id = $variables['attributes']['id'];
+    $id = str_replace("-", "_", $table_id);
+    $suggestions[] = 'table' . '__' . $id;
+  }
+}
+
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/pineapple.theme:91]: Remove this please.

File: web/themes/custom/pineapple/templates/dataset/table--edit-password-policy-status.html.twig [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{#
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/templates/dataset/table--edit-password-policy-status.html.twig:1]: Delete this file. There's no reason to not make every table on your site responsive.

File: web/themes/custom/pineapple/pineapple.theme (hunk 1 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/pineapple.theme]: done

File: web/themes/custom/pineapple/templates/dataset/table--edit-password-policy-status.html.twig (hunk 2 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/templates/dataset/table--edit-password-policy-status.html.twig]: Is what I've done okay?

Comment (ElXyfer) [reply web/themes/custom/pineapple/templates/dataset/table--edit-password-policy-status.html.twig]: Yes

**Pull request merged**